### ISSUE TITLE
[Feature Request] Show branch name in repo list

### ISSUE THREAD

**Issue by mtschoen-unity** [ForkIssues/Tracker#646]
Hi there! I just got the update with the new sidebar/repo browser view. Looks great!

Something I always wanted in the old view (and would still like it in this view) was to see the current branch of each repository. I have multiple copies of each project that I'm working on to do code reviews without having to stash and checkout what I am working on across multiple submodules. At first I tried to come up with helpful folder names and always use one for review and one for feature work, but it quickly devolved into just `project-name1`, `project-name2`, etc. It would be really helpful to know what repo was checked out to what branch when I came back to them. I currently have to open windows/tabs for each repo to check this.

**Comment by DanPristupov** [comment]
Hi. I started improving the repository manager and I'll think about showing the active branch name in the additional column. However performance is a big concern there.

**Comment by mtschoen-unity** [comment]
Yeah. Maybe you could do the branch check as a second, asynchronous check after the first load? Sounds like more work but it would be fine if it takes a while to fill out the branch names after getting the basic list.

**Comment by jvincent-siplay** [comment]
This would be great.  I'd also love the ability to right-click and copy the current branch URL.

**Comment by GeniusWiki** [comment]
This is probably the only missing feature from VSCode, I have to use VSCode to check the status of all my submodules. In addition to displaying the branch name, it is better to indicate if it has unstaged files. In the submodule of the root project, it would be great if the "submodule" list could also display this branch name and status.

VSCode screenshot:
[https://www.dropbox.com/s/p9u0ehj948jomoh/Screen%20Shot%202019-09-30%20at%2009.40.37.png?dl=0](https://www.dropbox.com/s/p9u0ehj948jomoh/Screen%20Shot%202019-09-30%20at%2009.40.37.png?dl=0)

**Comment by DanPristupov** [comment]
@GeniusWiki what do the stars mean? A dirty working directory or non-pushed changes?

**Comment by GeniusWiki** [comment]
@DanPristupov  Dirty working directory. Actually VSCode also has indicator for non-pushed. 

This tells dev branch, has 144 commits down, 0 up, very informative. I don't need to check one by one projects. 
https://www.dropbox.com/s/soencs9mrab6nua/Screen%20Shot%202019-09-30%20at%2014.52.32.png?dl=0
